draft-ietf-wimse-s2s-protocol.md

```diff
@@ -351,11 +354,12 @@ The decoded JWT claims of the WPT from the example above are shown here:
 
 ~~~ json
 {
- "iss": "wimse://example.com/specific-workload",
- "aud": "https://service.example.com/path",
- "exp": 1717612820,
- "jti": "__bwc4ESC3acc2LTC1-_x",
- "ath": "CL4wjfpRmNf-bdYIbYLnV9d5rMARGwKYE10wUwzC0jI"
+  "ath": "CL4wjfpRmNf-bdYIbYLnV9d5rMARGwKYE10wUwzC0jI",
+  "aud": "https://service.example.com/path",
+  "exp": 1728658672,
+  "iss": "wimse://example.com/specific-workload",
+  "jti": "4b42c5f611e2b1cfa1d2c41b3a2fb782",
```

Comment:
Whhhhhhyyyyyyyy?  Hex in jti values is so aesthetically displeasing.

Reply:
If you feel strongly about this I can provide a new PR that changes this identifier ;)

Reply:
I do kinda. `__bwc4ESC3acc2LTC1-_x` was okay. and the order of the claims was more readable before